docs/workload.rst

```diff
@@ -0,0 +1,193 @@
+
+          appliance_uuid: c74bca54-e903-49e8-9962-2bb895f428c1
+            cluster_name: example-cluster-name
+              created_at: 2021-02-25T04:54:41.362Z
+         datacenter_name: cwp-bucket-1-datacenter
+             eligibility: ELIGIBLE
+        eligibility_code: None
+           esx_host_name: 10.105.17.113
+           esx_host_uuid: a2311b42-3e53-8f21-97d7-66680007185f
+               host_name: appd2012
+                      id: 19902164
+     installation_status: NOT_INSTALLED
+    installation_status_code:
+              ip_address: 10.105.17.84
+                    name: cwp-bucket-1-windows_2012
+         os_architecture: 64
+          os_description: Microsoft Windows Server 2012 (64-bit)
+                 os_type: WINDOWS
+                    uuid: 500e14e6-3ea6-23aa-11bd-8e68444c6ce4
+        vcenter_host_url: 10.105.17.114
+            vcenter_name: VMware vCenter Server 6.7.0 build-14368073
+            vcenter_uuid: 9a8a0be5-ae1e-49ce-b2aa-34bc7dc445e3
+     vmwaretools_version: 11328 ComputeResource object, bound to https://defense-dev01.cbdtest.io.
+  -------------------------------------------------------------------------------
+
+          appliance_uuid: c74bca54-e903-49e8-9962-2bb895f428c1
+            cluster_name: example-cluster-name
+              created_at: 2021-02-25T04:54:41.362Z
+         datacenter_name: cwp-bucket-1-datacenter
+             eligibility: ELIGIBLE
+        eligibility_code: None
+           esx_host_name: 10.105.17.113
+           esx_host_uuid: a2311b42-3e53-8f21-97d7-66680007185f
+               host_name: appd2k8r2
+                      id: 19902168
+     installation_status: NOT_INSTALLED
+    installation_status_code:
+              ip_address: 10.105.17.237
+                    name: cwp-bucket-1-windows_2008
+         os_architecture: 64
+          os_description: Microsoft Windows Server 2008 R2 (64-bit)
+                 os_type: WINDOWS
+                    uuid: 500e51ff-ca0d-5a70-a799-2595c9e87000
+        vcenter_host_url: 10.105.17.114
+            vcenter_name: VMware vCenter Server 6.7.0 build-14368073
+            vcenter_uuid: 9a8a0be5-ae1e-49ce-b2aa-34bc7dc445e3
+     vmwaretools_version: 11328 ComputeResource object, bound to https://defense-dev01.cbdtest.io.
+
+
+Interactive example script featuring Workloads Search
+-------------------------------------------------------------------------------
+Another cool feature we provide with our python SDK are a number of example scripts.
+
+.. image:: _static/workloads_example_script.gif
+
+This interactive script, in particular, will show you the capabilities of our SDK in a clear and precise manner. It is designed in such a way, that everybody, regardless of technical skill or level can use it intuitively.
+
+You can download it from `here <https://github.com/carbonblack/carbon-black-cloud-sdk-python/blob/develop/examples/workload/workloads_search_example.py>`_
```

Comment:
should this link be formatted like: [here](https://github.com/carbonblack/carbon-black-cloud-sdk-python/blob/develop/examples/workload/workloads_search_example.py)